README.md

```diff
@@ -1,2 +1,377 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+    **Body Data**
+
+  ``` json
+  {
+    "email": "you@me.com"
+  }
+  ```
+
+## get user
+
+- **Success Response:**
+  - **Code:** 200
+  - **Content:**
+
+```json
+{
+  "message": "Successfully retrieved user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+- **Code:** 400 SERVER ERROR
+- **Content:** `{"message": "Invalid email", "data": {} }`
+
+- **Code:** 500 SERVER ERROR
+- **Content:** `{"message": "Unexpected error", "data": {} }`
+
+#### add user if not existing
+
+- **Success Response:**
+  - **Code:** 201
+  - **Content:**
+
+```json
+{
+  "message": "Successfully created user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+  - **Code:** 500 SERVER ERROR
+    **Content:** `{"message": "Unexpected error"}`
+
+## Return all recipes for a user - story3
+
+- **URL**
+
+  /users/:userId/recipes
+
+- **Method:**
+
+  `GET`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users/1/recipes`
+
+- **Success Response:**
+- **Code:** 200
+- **Content:**
+
+```json
+{
+  "message": "Successfully retrieved all recipes",
+  "data": [
+    {
+      "id": 1,
+      "name": "baps",
+      "duration": 3
+    },
+    {
+      "id": 2,
+      "name": "buns",
+      "duration": 3
+    }
+  ]
+}
+```
+
+- **Error Response:**
+
+  - **Code:** 400 BAD REQUEST
+    **Content:** `{"message": "Invalid user id", "data": [] }`
+
+  - **Code:** 500 SERVER ERROR
+    **Content:** `{"message": "Unexpected error", "data": [] }`
+
+## Add a recipe for a user - Story-3
+
+- **URL**
+
+  /users/:userId/recipes
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users/1/recipes` 
+
+  **Body Data**
+
+  Required
+
+  ``` json
+  {
+    "name": "string",
+    "instructions": "string",
+    "prep_time": 0,
+    "cook_time": 0,
+  }
```

Comment:
What if there are ingredients for this recipe too? Just going to figure it out when you get to story 4?